README.md

```diff
@@ -5,6 +5,8 @@
 An [express.js]( https://github.com/visionmedia/express ) middleware for
 [node-validator]( https://github.com/chriso/validator.js ).
 
+*Note: As of node-validator's v5, it does not automatically coerce values to strings, but it still requires strings as input. So now express-validator will automatically coerce your input to strings before to passes to node-validator. Ideally, `notEmpty()` should still function as normal.*
```

Comment:
This note may be unnecessary. The behavior (after some adjustments) should be just as it was previously, only that with the upgrade of [validator.js](https://github.com/chriso/validator.js) (note the new name), the string coercion is performed by express-validator rather than validator.js. The user need not be concerned with this detail since the interface has not changed.